ui/src/kapacitor/components/KapacitorRule.js

```diff
@@ -7,6 +7,7 @@ import RuleMessage from 'src/kapacitor/components/RuleMessage'
 import {createRule, editRule} from 'src/kapacitor/apis'
 import buildInfluxQLQuery from 'utils/influxql'
 import timeRanges from 'hson!../../shared/data/timeRanges.hson'
+import FancyScrollbox from 'src/shared/components/FancyScrollbar'
```

Comment:
Let's call it the same as its file not Box vs Bar.

Also, it should be imported next to components, a few lines up, in every file.